_layouts/default.html

```diff
@@ -21,58 +21,62 @@
+        document,'script','https://connect.facebook.net/en_US/fbevents.js');
+        fbq('init', '188598988238864');
+        fbq('track', 'PageView');
     </script>
-    <noscript><img height="1" width="1" style="display:none"
-    src="https://www.facebook.com/tr?id=188598988238864&ev=PageView&noscript=1"
-    /></noscript>
+    <noscript>
+        <img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=188598988238864&ev=PageView&noscript=1" />
+    </noscript>
     <!-- DO NOT MODIFY -->
     <!-- End Facebook Pixel Code -->
 
     <!-- Google Analytics -->
     <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-    ga('create', 'UA-44449737-1', 'auto');
-    ga('send', 'pageview');
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+        ga('create', 'UA-44449737-1', 'auto');
+        ga('send', 'pageview');
     </script>
 </head>
 <body>
     <div class="main-wrapper">
         <nav class="fixed-top full-width">
-            <div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium">
-                <a href="#home"><img src="{{ "img/logo.png" | prepend: site.baseurl }}?{{site.time | date: '%s%N'}}"></a>
+            <div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="large">
+                <a href="#home"><img src="{{ "img/logo.svg" | prepend: site.baseurl }}?{{site.time | date: '%s%N'}}"></a>
                 <div class="float-right">
-                    <div class="title-bar-title" data-toggle>Menu</div>
-                    <button class="menu-icon" type="button" data-toggle></button>
+                    <div class="title-bar-title" data-toggle="main-menu">Menu</div>
+                    <button class="menu-icon" type="button" data-toggle="main-menu"></button>
                 </div>
             </div>
 
             <div id="main-menu" class="top-bar">
                 <div class="row columns">
-                    <div class="top-bar-title show-for-medium">
-                        <a href="/"><img src="img/logo.png"></a>
+                    <div class="top-bar-title show-for-large">
+                        <a href="/"><img src="img/logo.svg"></a>
                     </div>
 
                     <div class="top-bar-left">
-                        <ul class="vertical medium-horizontal menu">
+                        <ul class="vertical large-horizontal menu">
```

Comment:
GitHub isn't letting me leave a line comment, but the Apply Now / Login links below probably also need the `large-horizontal` class

Reply:
👍